plugin-server/src/worker/ingestion/event-pipeline/populateTeamDataStep.ts

```diff
@@ -43,28 +36,8 @@ export async function populateTeamDataStep(
         token_present: event.token ? 'true' : 'false',
     })
 
-    // If a team_id is present (resolved at capture, or injected by an app), trust it but
-    // try resolving from the token if present, and compare results to detect inconsistencies.
-    // TODO: remove after lightweight capture is fully rolled-out and the
-    //       ingestion_event_hasauthinfo metric confirms all incoming events have a token.
+    // If a team_id is present (event captured from an app), trust it and return the event as is.
     if (event.team_id) {
```

Comment:
Do we want to still do `ip: team.anonymize_ips ? null : event.ip,` here to match the below and we can remove it from the later step?

Reply:
The current code is consistent with the "old" behaviour, so I'd keep it that way unless we prove it's needed.

[The `capture` we provide to apps](https://github.com/PostHog/posthog/blob/b04015f25ea0472c332f5c3b0c7fa108783fd4ec/plugin-server/src/worker/vm/extensions/posthog.ts#L37) does not allow passing an `ip` value, and hardcodes an empty one.

Reply:
I'd like it to be the same in the same step for consistency, it's not logical that we do this for one case and not for the other, I can't think of a reason why we'd want different behaviour here, can you? Can be a follow-up PR if you prefer.

Reply:
Agreed. Will address it in a follow-up PR where I'll refactor some of that logic with the session recordings consumer to avoid behaviours drifting